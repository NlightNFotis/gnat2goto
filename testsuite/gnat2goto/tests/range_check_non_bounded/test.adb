procedure Test is
   type Uint_8 is mod 2**8;
   Uint_Var : Uint_8 := 5;
   Int_Var : Integer;
begin
   Int_Var := Integer'(Integer (Uint_Var));
   pragma Assert (Int_Var = 5);
end Test;
