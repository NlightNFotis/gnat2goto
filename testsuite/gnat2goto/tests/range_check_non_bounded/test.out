[assertion.1] file test.adb line 6 Range Check: SUCCESS
[test.assertion.1] line 7 assertion : SUCCESS
VERIFICATION SUCCESSFUL
